core/src/test/java/org/elasticsearch/search/scriptfilter/ScriptQuerySearchIT.java

```diff
@@ -87,6 +103,57 @@ public Settings indexSettings() {
                 .build();
     }
 
+    public void testCustomScriptBinaryField() throws Exception {
+        final byte[] randomBytesDoc1 = getRandomBytes(15);
+        final byte[] randomBytesDoc2 = getRandomBytes(16);
+
+        assertAcked(
+                client().admin().indices().prepareCreate("my-index")
+                        .addMapping("my-type", createMappingSource("binary"))
+                        .setSettings(indexSettings())
+        );
+        client().prepareIndex("my-index", "my-type", "1")
+                .setSource(jsonBuilder().startObject().field("binaryData",
+                        Base64.getEncoder().encodeToString(randomBytesDoc1)).endObject())
+                .get();
+        flush();
+        client().prepareIndex("my-index", "my-type", "2")
+                .setSource(jsonBuilder().startObject().field("binaryData",
+                        Base64.getEncoder().encodeToString(randomBytesDoc2)).endObject())
+                .get();
+        flush();
+        refresh();
+
+        SearchResponse response = client().prepareSearch()
+                .setQuery(scriptQuery(
+                        new Script(ScriptType.INLINE, CustomScriptPlugin.NAME, "doc['binaryData'].get(0).length > 15", Collections.emptyMap())))
```

Comment:
This line is too long, it is causing checkstyle to fail. We have a line limit of 140 characters that we started enforcing a few months ago but only on files that didn't have any violations. We're super super slowly fixing the violations as we go.

I'll fix this before I merge.